server/apps/prepopulate/data_initialization/vocabularies.json

```diff
@@ -98,6 +98,29 @@
         ]
     },
     {
+        "_id": "priority",
```

Comment:
I removed the static list of priorities. Apart from that the list in vocabularies.json looks ok I guess.